### PR TITLE
NATIVE-25: Improve ImageView Modal

### PR DESCRIPTION
**Pull request by NKlug**

### CODE REVIEW

File: src/modules/layout/components/ModalHeader.js [hunk 1]
```diff
@@ -56,19 +41,16 @@ class Header extends React.PureComponent<PropsType, StateType> {
   }
 
   render () {
-    const headerTitle = this.getDescriptor().headerTitle || ''
-
     return (
       <BoxShadow theme={this.props.theme}>
         <Horizontal>
           <HorizontalLeft>
             <HeaderBackButton onPress={this.goBack} />
```
Comment (maxammann) [inline src/modules/layout/components/ModalHeader.js]: the null in goBack is acually required by design: https://reactnavigation.org/docs/en/navigation-prop.html#goback-close-the-active-screen-and-move-back

File: src/routes/image/components/ImageViewModal.js [hunk 2]
```diff
@@ -1,13 +1,21 @@
 // @flow
 
 import * as React from 'react'
-import { View } from 'react-native'
+import styled from 'styled-components'
 import { ImageViewer } from 'react-native-image-zoom-viewer'
 import type { NavigationScreenProp } from 'react-navigation'
+import type { ThemeType } from '../../../modules/theme/constants/theme'
+
+// adding a negative margin to also show the image beneath the header
```
Comment (maxammann) [inline src/routes/image/components/ImageViewModal.js]: This is very weird. This should not be necessary.

File: src/modules/layout/components/ModalHeader.js [hunk 3]
```diff
@@ -19,12 +19,6 @@ const HorizontalLeft = styled.View`
...
 `
 
-const Title = styled.Text`
- font-size: 30px;
- color: black;
- margin-left: 10px;
-`
-
 const BoxShadow = styled.View`
   background-color: transparent;
```
Comment (maxammann) [inline src/modules/layout/components/ModalHeader.js]: You can use:

```
const BoxShadow = styled.View`
  background-color: transparent;
  position: absolute;
  z-index: 100;
  top: 0;
  left: 0;
  right: 0;
  height: ${props => props.theme.dimensions.modalHeaderHeight};
`
```

That way the ModalHeader is always transparent and does not take space. You can also name this then TransparentHeader maybe. So it is more reusable :)

File: src/routes/image/components/ImageViewModal.js (hunk 2 above)
Comment (maxammann) [reply src/routes/image/components/ImageViewModal.js]: See the comment below. It actually is necessary and works as expected :)

File: src/modules/layout/components/ModalHeader.js (hunk 3 above)
Comment (NKlug) [reply src/modules/layout/components/ModalHeader.js]: Done. This somehow fixed the bug that the image is not displayed beneath the header. :+1:

File: src/modules/app/components/Navigator.js [hunk 4]
```diff
@@ -40,28 +40,42 @@ export const LandingStack = createSwitchNavigator(
...
 export default createStackNavigator(
   {
-    'LandingStack': LandingStack,
-    'ChangeLanguageModal': ChangeLanguageModalContainer,
-    'ModalStack': ModalStack,
-    'MapViewModal': MapViewModal,
-    'ImageViewModal': ImageViewModal
+    'LandingStack': {
+      screen: LandingStack,
+      navigationOptions: {
```
[the remark this answers is not in the document]
Comment (NKlug) [reply src/modules/app/components/Navigator.js]: Done.

**Pull request merged**